src/core/connection.c

```diff
@@ -4911,7 +4917,6 @@ QuicConnRecvDatagrams(
             Batch,
             Cipher,
             &RecvState);
-        BatchCount = 0;
```

Comment:
Again, I don't like removing lines like this that could result in bugs in the future.

Reply:
I think I'd rather NOLINT these type of things.